lib/nylas/raw_message.rb

```diff
@@ -11,8 +11,12 @@ def initialize(mime_compatible_string, api:)
     end
 
     def send!
-      Message.new(api.execute(method: :post, path: "/send", payload: mime_compatible_string,
-                              headers: { "Content-type" => "message/rfc822" }).merge(api: api))
+      Message.new(**api.execute(
+        method: :post,
+        path: "/send",
+        payload: mime_compatible_string,
+        headers: { "Content-type" => "message/rfc822" }
```

Comment:
Should this be a constant?

Reply:
Yes for sure. Done here 54360b6c3aefa6f531d1d3554757fd4e33cd58b0